spec/api_spec.rb

```diff
@@ -10,6 +10,19 @@ def expect_match(compare, base)
     expect(compare.parsed_response).to eq(base.parsed_response)
   end
 
+  def expect_match_all_but_assigned_user(compare_query_request, base_query_request)
```

Comment:
Could this method just accept the (already executed) response instead of a lambda? I feel like wrapping the request in a lambda is more complexity than needed here.